test/e2e/plain_provisioner_test.go

```diff
@@ -735,7 +737,94 @@ var _ = Describe("plain provisioner bundle", func() {
 				}).Should(BeNil())
 			})
 		})
+	})
+
+	When("the bundle is backed by a configmap", func() {
+		var (
+			bundle    *rukpakv1alpha1.Bundle
+			configmap *corev1.ConfigMap
+			ctx       context.Context
+		)
+
+		BeforeEach(func() {
+			ctx = context.Background()
+
+			data := map[string]string{}
+			err := filepath.Walk(filepath.Join(testdataDir, "bundles/plain-v0/valid/manifests"), func(path string, info os.FileInfo, err error) error {
+				if err != nil {
+					return err
+				}
+				if info.IsDir() {
+					return nil
+				}
+				c, err := ioutil.ReadFile(path)
+				if err != nil {
+					return err
+				}
+				data[info.Name()] = string(c)
+				return nil
+			})
+			Expect(err).To(BeNil())
+			configmap = &corev1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "local-configmap-",
+					Namespace:    defaultSystemNamespace,
+				},
+				Data: data,
+			}
+			err = c.Create(ctx, configmap)
+			Expect(err).To(BeNil())
+			bundle = &rukpakv1alpha1.Bundle{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "combo-local-",
+				},
+				Spec: rukpakv1alpha1.BundleSpec{
+					ProvisionerClassName: plain.ProvisionerID,
+					Source: rukpakv1alpha1.BundleSource{
+						Type: rukpakv1alpha1.SourceTypeLocal,
+						Local: &rukpakv1alpha1.LocalSource{
+							ConfigMapRef: &rukpakv1alpha1.ConfigMapRef{
+								Name:      configmap.ObjectMeta.Name,
+								Namespace: defaultSystemNamespace,
+							},
+						},
+					},
+				},
+			}
+			err = c.Create(ctx, bundle)
+			Expect(err).To(BeNil())
+		})
+
+		AfterEach(func() {
+			err := c.Delete(ctx, bundle)
+			Expect(client.IgnoreNotFound(err)).To(BeNil())
+		})
 
+		It("Can create and unpack the bundle successfully", func() {
```

Comment:
Would it make sense to add a test case for defining what should happen if the `Bundle` inside of the `ConfigMap` is invalid?

Reply:
Actually, since I'm BDC this week, I'd rather go ahead and merge this as-is and track follow-up e2e tests as a separate issue. This would avoid any potential merge conflicts.

Reply:
https://github.com/operator-framework/rukpak/issues/451